android/src/main/kotlin/com/oguzhnatly/flutter_carplay/FCPScreen.kt

```diff
@@ -21,10 +21,10 @@ class FCPScreen(carContext: CarContext, val fcpTemplate: FCPTemplate) : Screen(c
             when (fcpTemplate) {
                 is FCPInformationTemplate -> {
                     AndroidAutoService.session?.pop()
-                    FCPStreamHandlerPlugin.sendEvent(
-                        type = FCPChannelTypes.onInformationTemplatePopped.name,
-                        data = mapOf("elementId" to fcpTemplate.elementId)
-                    )
+                }
+
+                is FCPPresentTemplate -> {
+                    ndroidAutoService.session?.pop()
```

Comment:
Is there an A missing? ndroidAutoService?

Reply:
changed